core/libs/test_utils/src/lib.rs

```diff
@@ -340,24 +340,7 @@ pub fn assert_deleted_files_pruned(core: &Core) {
 
 /// Compare dbs for key equality don't compare last synced.
 pub fn assert_dbs_eq(left: &Core, right: &Core) {
-    assert!(keys_match(&left.db.account.get_all().unwrap(), &right.db.account.get_all().unwrap()));
-    assert!(keys_match(&left.db.root.get_all().unwrap(), &right.db.root.get_all().unwrap()));
-    assert!(keys_match(
-        &left.db.local_digest.get_all().unwrap(),
-        &right.db.local_digest.get_all().unwrap()
-    ));
-    assert!(keys_match(
-        &left.db.base_digest.get_all().unwrap(),
-        &right.db.base_digest.get_all().unwrap()
-    ));
-    assert!(keys_match(
-        &left.db.local_metadata.get_all().unwrap(),
-        &right.db.local_metadata.get_all().unwrap(),
-    ));
-    assert!(keys_match(
-        &left.db.base_metadata.get_all().unwrap(),
-        &right.db.base_metadata.get_all().unwrap(),
-    ));
+    assert!(dbs_equal(left, right));
```

Comment:
This is certainly nicer, but when a test is failing the prior version would tell us what database was unequal, which was a helpful starting point for debugging. There's also an earlier iteration in the history which would use `assert_eq!` which spits out both sides, intellij would then have a button that would allow you to `find difference`.

Reply:
Not sure exactly what the best way to write it while not repeating ourselves.

Reply:
Ah ok, makes sense. For now I think I should just revert to how it was before.

Reply:
cool